src/avram/define_attribute.cr

```diff
@@ -100,4 +100,57 @@ module Avram::DefineAttribute
       end
     end
   end
+
```

Comment:
There's a lot of duplication here between `attribute` and `file_attribute`. I tried extracting similar code to a separate macro, but can't make that work. I also thought about calling `attribute` inside that `file_attribute` macro, which does work, but it requires a lot of reorganisation. Before I start on that, I'd like some input first.

Reply:
Solved!